app/component/AirportCollectLuggageLeg.js

```diff
@@ -18,7 +18,7 @@ function AirportCollectLuggageLeg(props, { config }) {
     <div className="row itinerary-row">
       <div className="small-2 columns itinerary-time-column">
         <div className="itinerary-time-column-time">
-          {moment(props.leg.endTime).format('HH:mm')}
+          {moment(props.leg.end).format('HH:mm')}
```

Comment:
I think you need to use the legTime method here to parse the actual time.

Reply:
Right - I grepped 'moment(props.leg', found many of those and fixed them all.